tex/polyglossia.sty

```diff
@@ -272,12 +272,9 @@
     \ifbool{xpg@hyphenation@disabled}{%
       \xdef\xpg@lastlanguage{\the\csname l@#1\endcsname}%
     }{%
-      \ifluatex%
-        \cs_if_eq:cNF{l@#1}{\l@nohyphenation}{%
-          \xpg@set@language@luatex@iv{#1}%
-        }%
-      \fi%
-      \language=\csname l@#1\endcsname%
+      \cs_if_eq:cNF{l@#1}{\l@nohyphenation}{%
+          \xpg@set@hyphenation@patterns{#1}%
+      }%
```

Comment:
Now that line 280 has been removed, selecting `nohyphenation` language such as `korean` or `arabic` will not have `\language=...` declaration. For instance, korean text in the following file will have hyphenations:
```latex
\documentclass{article}
\usepackage{polyglossia}
\setmainlanguage{english}
\setotherlanguages{korean}
\newfontfamily\koreanfont{UnBatang}
\begin{document}
\hsize=1pt
english hyphenation
\textkorean{korean hyphenation}.
\end{document}
```

Reply:
Thanks, could you try again?

Reply:
Thanks. It has solved the issue.